<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculateGeometryWindowClass</class>
 <widget class="QDialog" name="CalculateGeometryWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>431</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>286</width>
    <height>390</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>286</width>
    <height>431</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>统计几何</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="4" column="1" rowspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>线段长</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>面积</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>输入图层：</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;注意：面要素计算周长和面积，&lt;/p&gt;&lt;p&gt;线要素计算长度，点不做计算&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLineEdit" name="shpArealineEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLineEdit" name="shpLinelineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>计算几何：</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_6">
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>请输入字段名：</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="startpushButton">
       <property name="text">
        <string>运行</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelpushButton">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
